webpack.config.js

```diff
@@ -118,7 +118,7 @@ const commonConfig = (env) => {
       alias: {
         "react": path.resolve('./node_modules/react'),
         "react-dom": path.resolve('./node_modules/react-dom'),
-        "@patternfly/react-core": path.resolve('./node_modules/@patternfly/react-core')
+        // "@patternfly/react-core": path.resolve('./node_modules/@patternfly/react-core')
```

Comment:
I forgot to remove this line from the previous PR, when we have the chance, let's :boom: it